include/extractor/extraction_way.hpp

```diff
@@ -46,6 +46,7 @@ struct ExtractionWay
         roundabout = false;
         circular = false;
         is_startpoint = true;
+        ignore_in_grid = false;
```

Comment:
Capturing from chat: Next actions:

- [ ] remove dead flag (ignore_in_grid) from foot/bicycle profile
- [ ] remove bindings for (ignore_in_grid)